<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/259453-14061G2132487.jpg);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>班级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>借阅册数</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="filterquerygroupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>过滤查询:　</string>
   </property>
   <widget class="QComboBox" name="columncomboBox">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>20</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>学号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>姓名</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>班级</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="orderlabel">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>根据()排序:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pricelineEdit">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>借阅册数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="increaseradioButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>207</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>升序</string>
    </property>
   </widget>
   <widget class="QComboBox" name="andorcomboBox">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>80</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&amp;&amp;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>||</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="columncomboBox2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>170</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>学号</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>姓名</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>班级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>借阅册数</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="biglabel">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>122</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="littlelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="descendradioButton">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>230</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>降序</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="biglineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="querypushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textlineEdit">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="operategroupBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>202</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>操作:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>178</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deletepushButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addpushButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="littlelineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QGroupBox" name="tablegroupBox">
    <property name="title">
     <string>表:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>254</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="storageradioButton">
        <property name="text">
         <string>仓库</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="importRecordradioButton">
        <property name="text">
         <string>导入数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="exportRecordradioButton">
        <property name="text">
         <string>导出数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="refreshStorageButton">
    <property name="text">
     <string>刷新仓库</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>178</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="closepushButton">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modifypushButton">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tableWidget</zorder>
  <zorder>filterquerygroupBox</zorder>
  <zorder>operategroupBox</zorder>
  <zorder>littlelineEdit</zorder>
  <zorder>splitter</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>refreshStorageButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closepushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
